benchmark_runner/common/template_operations/render_yaml_from_template.py

```diff
@@ -16,12 +16,14 @@ def render_yaml_string(template_str: str, environment_variable_dict: dict = None
     if not environment_variable_dict:
         environment_variable_dict = environment_variables.environment_variables_dict
     # Allow faking the Prometheus token so that we can generate deterministic test files
-    if 'prom_token_override' in environment_variable_dict:
-        environment_variable_dict['prom_token'] = environment_variable_dict.get('prom_token_override', '')
-    else:
-        oc = OC(kubeadmin_password=environment_variable_dict.get('kubeadmin_password', ''))
-        # set prom token
-        environment_variable_dict['prom_token'] = oc.get_prom_token()
+    if environment_variable_dict.get("enable_prometheus_snapshot", None):
```

Comment:
Just run:
if environment_variable_dict.get("enable_prometheus_snapshot"):
No need None

Reply:
It's not necessary, because `dict.get()` defaults the second argument to None.

Reply:
That's right, just removed the None.